client/src/components/AllTabs/AllTabs.css

```diff
@@ -0,0 +1,66 @@
+.Tabs {
+    width: 89%;
+    min-width: 1200px;
+  height: auto;
+  min-height: 400px;
+  margin-top: 20px;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 2rem 1rem;
+  color: black;
+  border-radius: 2rem;
+
+}
+
+.Tabs ul.nav {
+    width: 89%;
+    min-width: 1200px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border: 1px solid rgba(42, 63, 38, 0.65);
+  border-radius: 2rem;
+  padding-left: 0px;
+
+}
+
+.Tabs li {
+  border-radius: 0;
+}
+
```

Comment:
Random q but what is the default border-radius? Is it not 0?

Reply:
I think I had to specify this because for some reason the lis in the middle of the tab list were becoming rounded when I only wanted the first and last tab to be rounded.